### PR TITLE
Add KorNLU dataset

### PR DESCRIPTION
**Pull request by sumanthd17**
Added Korean NLU datasets. The link to the dataset can be found [here](https://github.com/kakaobrain/KorNLUDatasets) and the paper can be found [here](https://arxiv.org/abs/2004.03289)

**Note**: The MNLI tsv file is broken, so this code currently excludes the file. Please suggest other alternative if any @lhoestq 

- [x] Followed the instructions in CONTRIBUTING.md
- [x] Ran the tests successfully
- [x] Created the dummy data

### CODE REVIEW

File: datasets/kor_nlu/kor_nlu.py [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+            **kwrags: keyword arguments frowarded to super
+        """
+        super(KorNluConfig, self).__init__(version=datasets.Version("1.0.0", ""), **kwargs)
+        self.description = description
+        self.data_url = data_url
+        self.citation = citation
+        self.url = url
+
+
+class korNlu(datasets.GeneratorBasedBuilder):
```
Comment (lhoestq) [inline datasets/kor_nlu/kor_nlu.py:61]:
```suggestion
class KorNlu(datasets.GeneratorBasedBuilder):
```

File: datasets/kor_nlu/kor_nlu.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+    BUILDER_CONFIGS = [
+        KorNluConfig(name=name, description=_DESCRIPTION, data_url=_DATA_URLS[name], citation=_CITATAION, url=_URL)
+        for name in ["nli", "sts"]
+    ]
```
Comment (lhoestq) [inline datasets/kor_nlu/kor_nlu.py:65]:
```suggestion
    ]
    BUILDER_CONFIG_CLASS = KorNluConfig
```

File: datasets/kor_nlu/kor_nlu.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+    def _info(self):
+        features = {}
+        if self.config.name == "nli":
+            labels = ["entailment", "neutral", "contradiction"]
+            features["sentence1"] = datasets.Value("string")
+            features["sentence2"] = datasets.Value("string")
```
Comment (lhoestq) [inline datasets/kor_nlu/kor_nlu.py:72]: could you rename them `premise` and `hypothesis` for consistency with other NLI datasets ?

**Review by lhoestq — COMMENTED** [review]
Looking very good so far !

Could you just change the names of the features for the NLI config ? we try to always use the same column names for consistency

File: datasets/kor_nlu/kor_nlu.py (hunk 3 above)
Comment (sumanthd17) [reply datasets/kor_nlu/kor_nlu.py]: made this change

**Comment by lhoestq** [comment]
Nice thank you !
Can you regenerate the dataset_infos.json file ? Since we changed the features we must update it

Then I think we'll be good to merge :)

**Review by lhoestq — APPROVED** [review]
LGTM thanks !

**Pull request merged**